packages/playwright-core/src/server/electron/electron.ts

```diff
@@ -127,10 +127,15 @@ export class Electron extends SdkObject {
       const artifactsDir = await fs.promises.mkdtemp(ARTIFACTS_FOLDER);
 
       const browserLogsCollector = new RecentLogsCollector();
+      const env = options.env ? envArrayToObject(options.env) : process.env;
+      // When debugging Playwright test that runs Electron, NODE_OPTIONS
+      // will make the debugger attach to Electron's Node. But Playwright
+      // also needs to attach to drive the automation. Disable external debugging.
+      delete env.NODE_OPTIONS;
```

Comment:
Not sure if we always want to do that or only if it's used with the vsc extension.

Reply:
Debugging is mutually exclusive with driving Electron because they are using the same debugging channel.